### PR TITLE
<TimeInput/> - added showSeconds prop

### PR DESCRIPTION
**Pull request by israelabuhay**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: src/TimeInput/TimeInput.st.css [hunk 1]
```diff
@@ -31,6 +31,10 @@
   flex-direction: row-reverse;
 }
 
+.root:showSeconds .input::input {
+width: 73px;
```
Comment (gendeld) [inline src/TimeInput/TimeInput.st.css:35]:
```suggestion
  width: 73px;
```

File: src/TimeInput/docs/index.story.js [hunk 2]
```diff
@@ -112,6 +113,11 @@ export default {
             text: 'TimeInput supports 24h mode',
             source: examples.disableAmPm,
           }),
+          example({
+            title: 'With seconds',
+            text: 'TimeInput supports sowing seconds',
```
Comment (gendeld) [inline src/TimeInput/docs/index.story.js:118]:
```suggestion
            text: 'TimeInput supports showing seconds',
```

File: src/TimeInput/test/TimeInput.spec.js [hunk 3]
```diff
@@ -101,6 +113,31 @@ describe('TimeInput', () => {
...
+      const props = {
+        showSeconds: true,
+        defaultValue: defaultMoment,
+      };
+      const { driver } = render(<TimeInput {...props} />);
+
+      expect(await driver.getValue()).toBe(
+        formatTime(props.defaultValue, `${time12Hours}${withSeconds}`),
+      );
+    });
```
Comment (gendeld) [inline src/TimeInput/test/TimeInput.spec.js:126]: Add:
```
expect(await driver.isShowSeconds()).toBe(true);
```

File: src/TimeInput/test/TimeInput.spec.js [hunk 4]
```diff
@@ -101,6 +113,31 @@ describe('TimeInput', () => {
...
+      const props = {
+        showSeconds: true,
+        defaultValue: defaultMoment,
+        disableAmPm: true,
+      };
+      const { driver } = render(<TimeInput {...props} />);
+
+      expect(await driver.getValue()).toBe(
+        formatTime(props.defaultValue, `${time24Hours}${withSeconds}`),
+      );
```
Comment (gendeld) [inline src/TimeInput/test/TimeInput.spec.js:138]: Add:
```
expect(await driver.isShowSeconds()).toBe(true);
```

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6186

**Pull request merged**